modules/Bio/EnsEMBL/Analysis/Hive/Config/OtherFeatureDb.pm

```diff
@@ -0,0 +1,583 @@
+    exonerate_protein_cov => 50,
+    exonerate_max_intron => 200000,
+
+# RNA-seq pipeline stuff
+    # You have the choice between:
+    #  * using a csv file you already created
+    #  * using a study_accession like PRJEB19386
+    #  * using the taxon_id of your species
+    # 'rnaseq_summary_file' should always be set. If 'taxon_id' or 'study_accession' are not undef
+    # they will be used to retrieve the information from ENA and to create the csv file. In this case,
+    # 'file_columns' and 'summary_file_delimiter' should not be changed unless you know what you are doing
+
+    # If your reads are unpaired you may want to run on slices to avoid
+    # making overlong rough models.  If you want to do this, specify a
+    # slice length here otherwise it will default to whole chromosomes.
+    slice_length => 10000000,
+
+## Add in genewise path and put in matching code
+    'genewise_pid'                        => '50',
+    'genewise_cov'                        => '50',
+    'genewise_region_padding'             => '50000',
+    'genewise_calculate_coverage_and_pid' => '1',
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+########################################################
+# URLs for retrieving the INSDC contigs and RefSeq files
+########################################################
+    'ncbi_base_ftp'           => 'ftp://ftp.ncbi.nlm.nih.gov/genomes/all',
+    'insdc_base_ftp'          => $self->o('ncbi_base_ftp').'/#expr(substr(#assembly_accession#, 0, 3))expr#/#expr(substr(#assembly_accession#, 4, 3))expr#/#expr(substr(#assembly_accession#, 7, 3))expr#/#expr(substr(#assembly_accession#, 10, 3))expr#/#assembly_accession#_#assembly_name#',
+    'assembly_ftp_path'       => $self->o('insdc_base_ftp'),
+    'refseq_base_ftp'         => $self->o('ncbi_base_ftp').'/#expr(substr(#assembly_refseq_accession#, 0, 3))expr#/#expr(substr(#assembly_refseq_accession#, 4, 3))expr#/#expr(substr(#assembly_refseq_accession#, 7, 3))expr#/#expr(substr(#assembly_refseq_accession#, 10, 3))expr#/#assembly_refseq_accession#_#assembly_name#',
+    'refseq_import_ftp_path'  => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_genomic.gff.gz',
+    'refseq_mrna_ftp_path'    => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_rna.fna.gz',
+    'refseq_report_ftp_path' => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_assembly_report.txt',
+##################################
+# Memory settings for the analyses
+##################################
+    'default_mem'          => '900',
+    'genblast_mem'         => '1900',
+    'genblast_retry_mem'   => '4900',
+    'genewise_mem'         => '3900',
+    'genewise_retry_mem'   => '5900',
+    'refseq_mem'           => '9900',
+    'projection_mem'       => '1900',
+    'layer_annotation_mem' => '3900',
+    'genebuilder_mem'      => '1900',
```

Comment:
These are parameters for different pipelines which are not needed here, they can be removed